engine/src/flutter/testing/assert_ok.h

```diff
@@ -0,0 +1,26 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_TESTING_ASSERT_OK_H_
+#define FLUTTER_TESTING_ASSERT_OK_H_
+
+#include "gtest/gtest.h"
+
+// Asserts that the given absl::Status or absl::StatusOr is OK.
+// If it is not OK, the status error message is printed.
+#ifdef ASSERT_OK
```

Comment:
Abseil provides matchers for `absl::StatusOr`:

```
#include "third_party/abseil-cpp/absl/status/status_matchers.h"
using ::absl_testing::IsOk;

ASSERT_THAT(stages, IsOk());
```

The latest Abseil also defines `ABSL_ASSERT_OK`/`ABSL_EXPECT_OK` macros similar to this header
(see https://github.com/abseil/abseil-cpp/commit/2487de07)

These macros landed shortly after Flutter's most recent Abseil roll.  I can do another roll of Abseil to get this update.